.travis.yml

```diff
@@ -42,3 +42,5 @@ branches:
   only:
     - master
     - /^greenkeeper/.*$/
```

Comment:
Can you remove `greenkeeper`?

Reply:
@nabdelgadir I think Miroslav removed it in his PR, so when I rebase again, it should be moved.